common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -256,19 +259,17 @@ private static LogLevel adapt(com.microsoft.identity.common.logging.Logger.LogLe
         }
     }
 
-    private void logDeprecationWarningNonStatic() {
-        if (sLogDeprecationWarning) {
-            sLogDeprecationWarning = false;
-            com.microsoft.identity.common.logging.Logger.warn(TAG, "This class is deprecated. "
-                    + "Migrate usage to: com.microsoft.identity.common.logging.Logger");
+    private void emitDeprecationEventNonStatic() {
+        if (sEmitDeprecationEvent) {
+            sEmitDeprecationEvent = false;
+            Telemetry.emit(new DeprecatedApiUsageEvent().putDeprecatedClassUsage(Logger.class));
         }
     }
 
-    private static void logDeprecationWarning() {
-        if (sLogDeprecationWarning) {
-            sLogDeprecationWarning = false;
-            com.microsoft.identity.common.logging.Logger.warn(TAG, "This class is deprecated. "
-                    + "Migrate usage to: com.microsoft.identity.common.logging.Logger");
+    private static void emitDeprecationEvent() {
```

Comment:
I believe you can can just call this static method everywhere and delete the other Non static method.

Reply:
@AdamBJohnsonx  why did you add the non static method?

Reply:
> @AdamBJohnsonx why did you add the non static method?

This was to resolve a spotBugs warning.  You could just as well add an @SuppressFBWarnings annotation.